package.json

```diff
@@ -13,10 +14,14 @@
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"build": "rollup -c build/rollup.config.js",
+		"lint": "xo lib test",
+		"test": "yarn build && yarn lint && yarn test:lib && yarn test:dist",
+		"test:lib": "ava test/lib.test.js",
+		"test:dist": "ava test/dist.test.js"
```

Comment:
The dist file should be built at the `prepublish` step. The dist files should not be committed.

Reply:
The dist files are not commited afaik as `dist` is in .gitignore?